### ISSUE TITLE
RAM 3.7 thinks that chromium is corrupted

### ISSUE THREAD

**Issue by SuperGamingBros4** [ic3w0lf22/Roblox-Account-Manager#305]
As the title says, Roblox Account Manager 3.7 thinks that chromium is corrupted when other versions don't.
I think that this may be an issue with windows 10, as all of the computers where I've seen it open the login page on are windows 11 PCs.

**Comment by SuperGamingBros4** [comment]
This issue also appears on 3.6.x version of Roblox Account Manager.

**Comment by Karunos** [comment]
i have a similar issue when trying to add more accounts, it opens chromium but it just loads for a minute or so and then says, this website is not avalible and the follows with Roblox.com took to long to respond or something like that, does that happen to you guys aswell and if so did u fix it and if so then how?

**Comment by Cbo11** [comment]
yes, I have the same issue except I'm on windows 11. it says this: 
![image](https://github.com/ic3w0lf22/Roblox-Account-Manager/assets/123276421/99b68c4f-2925-4053-9b2d-1714319f4f80)
but when i click yes it just tells me to close the app, i wait and open it again and it says 'installing chromium' then i click add account and it does this:
![image](https://github.com/ic3w0lf22/Roblox-Account-Manager/assets/123276421/0e74daed-6474-4f77-8495-7aa68738037e)
